### PR TITLE
OCM-11496 | fix: Change language when updating cluster reg config

### PR DESCRIPTION
**Pull request by patrickjennings**
https://issues.redhat.com/browse/OCM-11496

### CODE REVIEW

File: pkg/clusterregistryconfig/flags.go [hunk 1]
```diff
@@ -147,12 +147,17 @@ func GetClusterRegistryConfigOptions(cmd *pflag.FlagSet,
 	}
 
 	if interactive.Enabled() && isHostedCP {
+		regConfigQuestion := "Enable registries config"
```
Comment (philipwu08) [inline pkg/clusterregistryconfig/flags.go:150]: Maybe extract the next few lines to a separate function so that we can add unit tests for it.

Comment (patrickjennings) [reply pkg/clusterregistryconfig/flags.go]: Let me know if the changes I pushed are what you are expecting.

File: pkg/clusterregistryconfig/flags_test.go [hunk 2]
```diff
@@ -37,12 +38,12 @@ var _ = Describe("Cluster Registry Config tests", func() {
 				Equal("-----BEGIN CERTIFICATE-----\n/abc\n-----END CERTIFICATE-----"))
 
 		})
-		It("KO: fail if the spec file is invalid", func() {
+		It("OK: fail if the spec file is invalid", func() {
```
Comment (chenz4027) [inline pkg/clusterregistryconfig/flags_test.go:41]: Oh @patrickjennings it's not a typo. KO is for tests with expected failures while OK is for best case scenario. No need to change those.

Comment (patrickjennings) [reply pkg/clusterregistryconfig/flags_test.go]: Sorry about that. "KO", nice convention! Reverted those changes.

Comment (chenz4027) [reply pkg/clusterregistryconfig/flags_test.go]: Np! Also it's conventional to have only one commit per each change. CS side has the option to squash the commits but I don't think rosa has it.
You can do 
```
git rebase -i HEAD~2
```
comment the 2nd commit
And then force push the PR.

Comment (patrickjennings) [reply pkg/clusterregistryconfig/flags_test.go]: Gotcha. Rebased and squashed the commits.

Comment (chenz4027) [reply pkg/clusterregistryconfig/flags_test.go]: Perfect, thanks @patrickjennings

**Pull request merged**